database/migrations/2022_11_09_013917_update_files_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
These were both manual DB changes that I made to add functionality in the file browser.

Reply:
Ideally, all DB changes should be done with a migration so it can be repeatable between environments and new deploys.

Reply:
Yeah, I get this now. I wasn't as familiar with Laravel when I did this :)